types/stripe-js/payment-intents.d.ts

```diff
@@ -118,6 +118,15 @@ declare module '@stripe/stripe-js' {
     };
   }
 
+  interface CreatePaymentMethodGrabPayData extends PaymentMethodCreateParams {
```

Comment:
Can a `grabpay` PaymentMethod be created directly? i.e. without going through a PaymentIntent.

The API ref makes it seem like that is the case, though I know sometimes the API ref is misleading. 

In Stripe.js land, that means that someone could also write

```js
stripe.createPaymentMethod({
  type: 'grabpay',
  grabpay: {},
})
```
If this is a legitimate use case, the `CreatePaymentMethodGrabPayData` needs to be added to the `CreatePaymentMethodData` enum above.

Reply:
A `grabpay` PM doesn't hold any data, but we still do allow users to create one directly. Fixed.